### PR TITLE
Add page styles based on the new bootstrap forms

### PR DESCRIPTION
**Pull request by clarkepaul**
Supersedes https://github.com/silverstripe/silverstripe-cms/pull/1533 
Requires https://github.com/silverstripe/silverstripe-framework/pull/5826 and https://github.com/silverstripe/silverstripe-framework/pull/5832
Fixes https://github.com/silverstripe/silverstripe-cms/issues/1526

image

### CODE REVIEW

**Comment by clarkepaul** [comment]
@tractorcow I managed to transfer all the styles on to your recent bootstrap form work but I've got two things which I couldn't figure out.

1. Empty div holds the space if "top level" is chosen.
2. Missing "Step 1" title (will make comment inline)

File: code/controllers/CMSPageAddController.php [hunk 1]
```diff
@@ -85,8 +75,21 @@ public function AddForm() {
...
+				DBField::create_field(
+					'HTMLFragment',
+					sprintf($numericLabelTmpl, 2, _t('CMSMain.ChoosePageType', 'Choose page type'))
+				),
+				$pageTypes,
+				'Page'
 			)
 		);
+
+		// TODO Insert page step title
```
Comment (clarkepaul) [inline code/controllers/CMSPageAddController.php]: @tractorcow This was the code for the title I had working at one stage but no longer works, you think you can fix it?

**Comment by tractorcow** [comment]
Will look at this tomorrow.

**Comment by tractorcow** [comment]
I've rebased this and added a fix for the field label.

I've added some notes on https://github.com/silverstripe/silverstripe-framework/pull/5832

**Comment by clarkepaul** [comment]
Tested on chrome / IE and everything is good.

**Comment by clarkepaul** [comment]
Success and error messages still require work which is captured here https://github.com/silverstripe/silverstripe-framework/issues/5674

**Pull request merged**